packages/sei-cosmwasm/src/proto_structs.rs

```diff
@@ -8,6 +8,7 @@ use cosmwasm_std::{Decimal, Uint64};
 pub struct OracleExchangeRate {
     pub exchange_rate: Decimal,
     pub last_update: Uint64,
+    pub last_update_timestamp: u64,
```

Comment:
Any reason this isn't a Uint64

Reply:
because Uint64 is a string representation of a Uint64, and corresponding to that, the value for lastUpdate is sdk.Int on the chain side. However, on the chain side, we convert the block time to UnixMilli() which returns a primitive so we need it to be u64 to parse the numeric json field

Reply:
Got it